### ISSUE TITLE
Taint tracking doesn't work for .NET Core

### ISSUE THREAD

**Issue by KSchlobohm** [dotnet/roslyn-analyzers#5095]
### Analyzer

**Diagnostic ID**: [CA3006](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca3006): `Review code for process command injection vulnerabilities`

### Analyzer source

**NuGet Package**: [Microsoft.CodeAnalysis.NetAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers)

**Version**: 5.0.3 (Latest)

### Describe the bug

*CA3006* does not trigger in .NET Core.

### Steps To Reproduce

Provide the steps to reproduce the behavior:
1. Create a new ASP.NET Core MVC project
2. Add the sample code from documentation that demonstrates how to trigger the error
```
   public class HomeController : Controller
   {

        ...omitted...

        public IActionResult Index()
        {

            try
            {
                string input = Request.Form["in"];
                var p = Process.Start(input);
            }
            catch
            {

            }

            return View();
        }

```

Example repro - https://github.com/KSchlobohm/ExploringCodeAnalysisRules

### Expected behavior
![image](https://user-images.githubusercontent.com/23503973/117854262-f276d780-b24e-11eb-992a-eeab66a11bdb.png)
Code is treated as error (due to ruleset change)

### Actual behavior
![image](https://user-images.githubusercontent.com/23503973/117854352-07536b00-b24f-11eb-9aa6-db43e620cdd9.png)
No errors are reported. (validated that the .editorconfig was being applied to trigger this as warning)

## Additional context

Documentation that demonstrates how to trigger this issue may need to be updated.

**Comment by KSchlobohm** [comment]
Other examples stemming from (I think) the same root cause are:
* CA3009
* CA3011
* CA3012

**Comment by KSchlobohm** [comment]
@dotpaul , what are some of the sinks where these analyzers are active? I only tried the example from the doc and wasn't sure what other samples would be valid.

**Comment by dotpaul** [comment]
> @dotpaul , what are some of the sinks where these analyzers are active? I only tried the example from the doc and wasn't sure what other samples would be valid.

https://github.com/dotnet/roslyn-analyzers/blob/c0b0d1f02335c4b50f6e64783645562fed10f49a/src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataConfig.cs#L312 is a good place to start. The SinkInfos classes referenced are in the same directory.

**Comment by randomnoise** [comment]
I gave it a shot with #5106, i hope it's in the right direction. 😏

**Comment by danmoseley** [comment]
Should this be closed now?

**Comment by randomnoise** [comment]
PR is merged to both 2.9.x and main branches. If there is no extra work for the issue, i think it can be closed. 

Linking PR with the issue didn't work for non-main branch, I think that's the only reason.
